File: sdk/highlight-java/highlight-common/src/main/java/io/highlight/sdk/common/HighlightSessionId.java
package io.highlight.sdk.common;

/**
 * The HighlightSessionId defines a method to retrieve a sessionId witch
 * represents a currently session.
 */
public interface HighlightSessionId {

	/**
	 * Returns the current sessionId.
	 * 
	 * @return the sessionId as a string
	 */
	String sessionId();
}

File: sdk/highlight-php/src/SDK/Common/HighlightSessionId.php
<?php

declare(strict_types=1);

namespace Highlight\SDK\Common;

/**
 * The HighlightSessionId defines a method to retrieve a sessionId which
 * represents the current session.
 */
interface HighlightSessionId
{
    /**
     * Returns the current sessionId.
     *
     * @return string the sessionId as a string
     */
    public function sessionId(): string;
}
